render.c

```diff
@@ -116,7 +116,7 @@ void render_frame(struct swaylock_surface *surface) {
 	float type_indicator_border_thickness =
 		TYPE_INDICATOR_BORDER_THICKNESS * surface->scale;
 
-	if (state->args.show_indicator && state->auth_state != AUTH_STATE_IDLE) {
+	if (state->args.show_indicator && (state->auth_state != AUTH_STATE_IDLE || state->args.indicator_idle_visible)) {
```

Comment:
style: can you wrap this line so it's under 80 columns?

Reply:
Ah thanks my bad, wrapped the line.